language/tools/move-package/src/resolution/dependency_graph.rs

```diff
@@ -684,22 +743,24 @@ impl DependencyGraph {
         external_requests: &mut Vec<ExternalRequest>,
         progress_output: &mut Progress,
     ) -> Result<()> {
-        let pkg = match self.package_table.entry(name) {
-            Entry::Vacant(entry) => entry.insert(pkg),
+        let pkg = match self.package_table.get(&name) {
+            None => pkg,
```

Comment:
ditto to the comment above -- could you elaborate on why you needed to move away from the `entry` API?  I may be missing a detail here, but this case I think could also still use `entry`.

Reply:
I was hitting some problems related to borrowing and at some point I gave up. If we decide to use the current solution and this code stays, I will try harder!!!

Reply:
I went ahead and changed it back. Clearly this was some problem mid-hacking as it was a rather easy change...